### DISCUSSION TITLE
Ferrum Rocks!

### DISCUSSION THREAD

**Discussion by grahamotte** [rubycdp/ferrum#188] [Show and tell]
Took this screenshot the other day after switching a deployment from Selenium to Ferrum.

<img width="770" alt="Screen Shot 2021-08-17 at 10 10 32 AM" src="https://user-images.githubusercontent.com/6063967/129980944-194b5f35-9c5d-4781-a7f7-88f6b4b4b4aa.png">

I think the results speak for themselves. Great work everyone! 🎉

**Comment by route** [comment]
Great to hear! Thank you! :)
